app/internal/wm/GioView.java

```diff
@@ -132,6 +134,66 @@ private void setCursor(int id) {
 		setPointerIcon(pointerIcon);
 	}
 
+	enum Bar {
+      NAVIGATION,
+      STATUS
+    }
+
+    private void setBarColor(Bar t, int color, int luminance) {
+        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.LOLLIPOP) {
+            return;
+        }
+
+        Window window = ((Activity) this.getContext()).getWindow();
+
+        int insetsMask = 0;
+        int viewMask = 0;
```

Comment:
Drop the initializers ("= 0") and let the Java compiler ensure they're always set.